### PR TITLE
Added Sample Codes for API:Checktoken

### PR DESCRIPTION
**Pull request by zaycodes**
Hello @brendajerop, https://www.mediawiki.org/wiki/User:Zaycodes/Sandbox/API:Checktoken

### CODE REVIEW

File: javascript/README.md [hunk 1]
```diff
@@ -91,6 +91,8 @@ Code snippets in Javascript demonstrating how to use various modules of the [Med
   * [tag.js](tag.js): Remove the spam tag from log entry ID 123 with the reason Wrongly applied
 * [API:Duplicatefiles](https://www.mediawiki.org/wiki/API:Duplicatefiles)
   * [duplicate_files.js](duplicate_files.js): list duplicates of the given files
+* [API:Checktoken](https://www.mediawiki.org/wiki/API:Checktoken)
+  * [check_token.js](check_token.js): Get request to check a CSRF token
```
Comment (brendajerop) [inline javascript/README.md]: Make this `Check a CSRF token`.

Comment (zaycodes) [reply javascript/README.md]: Fixed

File: modules.json [hunk 2]
```diff
@@ -675,5 +678,16 @@
 			"totitle":"Template:UnsignedIP",
 			"format":"json"
 		}
+	},
+	{
+		"filename": "check_token",
+		"docstring": "Demo of `Checktoken` module: Get request to Check a CSRF token.",
```
Comment (brendajerop) [inline modules.json]: It seems that I can't see the change? The description is still the same

Comment (zaycodes) [reply modules.json]: @brendajerop , I've fixed it

**Pull request merged**